chidioguejiofor/maintenance-react-app
# Chore/160953539/configure webpack and jest

#### What does this PR do?
- updates webpack config
- update jest config
- creates a public folder where files would be stored

#### Description of Task to be completed?
- configure webpack and jest
- change folder structure

#### How should this be manually tested?
- npm test command should run test
- npm start should start the static server
- npm run dev should run the webpack-dev-server

#### Any background context you want to provide?
- none

#### What are the relevant pivotal tracker stories?
- 160953539

#### Screenshots (if appropriate)
- none

#### Questions:
-none
